models/in_tcpd_elections/code/upload.py

```
@@ -0,0 +1,104 @@
+        )
+    return n
+
+
+def main():
+    only = set(sys.argv[1:])
+    tables = [t for t in TABLES if not only or t[1] in only]
+    for dataset_id, table_id, expected in tables:
+        print(f"=== {dataset_id}.{table_id} ===", flush=True)
+        try:
+            upload_table(dataset_id, table_id, expected)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)